doc/content/examples/example3b.md

```diff
@@ -0,0 +1,38 @@
+## Example 3b: Nonlinear Site-Response Analysis explicit time integration
+
+[Example 3a](examples/example3a.md) can also be run using an explicit time integration through the `CentralDifference`
+scheme. Each iteration in explicit integration has only one linear solve and no nonlinear
+solves. This feature makes explicit integration always converge, unlike the implicit
+integration schemes. However, for explicit integration to converge, the timestep
+should be less than the so called critical time step. [`CentralDifference`](source/timeintegrators/CentralDifference.md)
+ and [`CriticalTimeStep`](source/postprocessors/CriticalTimeStep.md)
+ provide more information on explicit integration and its convergence, respectively.   
+
+Some modifications need to be made to the input file to use the `CentralDifference` scheme.
+Actions cannot be used in conjunction with `CentralDifference` at this time.
+Kernels, AuxKernels, material stress, and material strain blocks have to be included separately.
+`InertialForce` blocks does not require the `NewmarkBeta` parameters. `AuxKernels` for
+accelerations and velocities should be of the type `TestNewmarkTI` and not `NewmarkAccelAux`
+or `NewmarkVelAux`. Displacement input has to be applied to the soil column instead
+of an acceleration input. While the user can specify the time step for the solver,
+it is recommended to use the `CriticalTimeStep` postprocessor which automatically
+calculates the optimal time step based upon the material properties. A factor less
+than unity is recommended to be multiplied to the critical time step to ensure convergence.
+The input file is presented below:
+
+!listing examples/ex03b/shear_beam_Isoil_free_field_explicit.i
+
+The plot below compares the accelerations recored at the top of the soil column between
+explicit and implicit integration schemes. These results match closely.
+
+!media media/examples/Exp_Imp_ex03.png
```

Comment:
Also present the response spectra. Differences are hard to judge from response histories.

Reply:
Format the figures properly. Also use semi-log, not log-log.